code/game/machinery/doors/windowdoor.dm

```diff
@@ -25,6 +25,7 @@
 	density = 0
 	if(health == 0)
 		playsound(src, "shatter", 70, 1)
+	electronics = null

```

Comment:
If the electronics are kept inside the door, they need to be ejected or qdel'd before the reference is nulled out.